requirements.txt

```diff
@@ -48,6 +48,7 @@ pynsive==0.2.6
 pyparsing==2.1.10
 python-dateutil==2.6.1
 pytz==2017.3
+requests-jwt==0.5.3
```

Comment:
This is a bit concerning: 
https://github.com/tgs/requests-jwt/blob/master/README.rst#this-package-is-not-actively-maintained-and-has-never-had-a-security-audit

Reply:
the newest version was released about a month ago, and was updated 4 months ago. I can reach out to the maintainer and ask if that banner is old cruft from when it wasn't maintained.

Reply:
So the maintainer says he only added that to say he's not actively using this project anymore, and isn't working with jwt anymore, but he would provide minor fixes if needed.
This particular package is doing a very small piece of what we need (taking a secret and providing a hash of it in conjunction with "requests" lib.

I'd have to see what else is out there before ripping it out  to replace this functionality.

Reply:
Per our conversation in the meeting today, we will move forward with this, and I'll work with Andrew to implement an even better solution utilizing auth0 authorization mechanism for machine to machine communication.